kolibri/plugins/facility_management/assets/src/views/UserPage/index.vue

```diff
@@ -48,11 +43,32 @@
       </template>
     </UserTable>
 
+    <nav>
+      <span>{{ $tr('pagination', { visibleStartRange, visibleEndRange, numFilteredUsers }) }}</span>
```

Comment:
add a `dir="auto"` attribute to fix the way numbers appear in RTL mode. Also add a little bit of margin against the two UIBUttons.

Reply:
A little bit of space _between_ the buttons might be nice too.

Reply:
You can copy whatever the margin values are from ClassEnrollForm.vue. And if you add spacing between the buttons, we should copy those over to ClassEnrollForm form for consistency.

Reply:
Thanks @jonboiser I added some spacing to both files and added a dir attr to the buttons as well to more easily target the different styles with appropriate button margins. By putting the margins on the buttons, the label text is also spaced apart better from the buttons now.